index.js

```diff
@@ -3,20 +3,26 @@
 const hof = require('hof');
 const app = require('express')();
 const churchill = require('churchill');
-
 const router = require('./lib/router');
 const serveStatic = require('./lib/serve-static');
 const sessionStore = require('./lib/sessions');
 const settings = require('./lib/settings');
 const defaults = require('./lib/defaults');
 
-const getConfig = config => Object.assign({}, defaults, config);
+const getConfig = function (options) {
+  let assigned;
+  [].slice.call(arguments).forEach((arg) => {
+    assigned = Object.assign({}, defaults, arg);
+  })
+  return assigned
+}
```

Comment:
`assigned` is overwritten for each argument. When you have `getConfig({a: 1}, {b:2 })` then `assigned` ultimately contains only `Object.assign({}, defaults, {b:2})`. The reference to `{a: 1}` is lost.

An alternative is to use something like:

```js
const args = [].slice.call(arguments);
return Object.assign.apply(null, [{}, defaults].concat(args));
```